test/plugins/features/movies/index.test.js

```diff
@@ -84,11 +128,61 @@ describe('movies integration', () => {
           'id',
           'title',
           'release_year',
+          'locations',
           'object'
         ]);
       });
     });
 
   });
 
+  describe('addLocation', () => {
+
+    it('location to movie', () => {
+      return Bluebird.all([
+        new Movie({ title: MOVIE_LIST[2] }).fetch(),
+        new Location({ name: LOCATION_LIST[0] }).fetch()
+      ])
+      .spread((movie, location) => {
+        return Movies.inject({
+          url: `/movies/${movie.id}/locations`,
+          method: 'POST',
+          payload: { id: location.id }
+        });
+      })
+      .then((response) => {
+        expect(response.statusCode).to.eql(200);
+      });
+    });
+
+    it('adds a location to a non-existent movie', () => {
+      return new Location({ name: LOCATION_LIST[0] }).fetch()
+      .then((location) => {
+        return Movies.inject({
+          url: `/movies/9999/locations`,
+          method: 'POST',
+          payload: { id: location.id }
+        });
+      })
+      .then((response) => {
+        expect(response.statusCode).to.eql(404);
+      });
+    });
+
+    it('adds a non-existent location to a movie', () => {
```

Comment:
`errs when passed a non-existent location`

same as above, this should be a controller test